packages/opc-styles/typography/_headings.scss

```diff
@@ -1,70 +1,63 @@
-$heading--xxl:                3rem;    /// 48px
-$heading--xl:                 2.5rem;  /// 40px
-$heading--one:                2.25rem; /// 36px
-$heading--two:                1.75rem; /// 28px
-$heading--three:              1.5rem;  /// 24px
-$heading--four:               1.25rem; /// 20px
-
 @mixin heading--xxl {
   font: {
     family: var(--opc-global--Heading--font-Family, 'Red Hat Display');
-    size: $heading--xxl;
-    weight: 400;
+    size: $opc-global--FontSize--6xl;
```

Comment:
@gisumit can u see the inconsistency here, we are saying @mixin heading-xxl and providing the value as 6xl

Reply:
also i dont think we need to use 6xl for headings

Reply:
fixes